salt/files/cloud.profiles.d/proxmox.conf

```diff
@@ -0,0 +1,19 @@
+# This file managed by Salt, do not edit by hand!!
+{% set cloud = salt['pillar.get']('salt:cloud', {}) -%}
+{% if 'proxmox_profile_defaults' in cloud %}
+{% set defaults = cloud.get('proxmox_profile_defaults') %}
+
+prox-test-1:
+    provider: proxmox-test
+    image: ISO:vztmpl/debian-7.0-standard_7.0-2_i386.tar.gz
+    technology: openvz
+    host: testvm01
+    cpus: 2
+    memory: 2048
+    swap: 2048
+    disk: 5
+    password: {{ defaults.get('default_password', 'password') }}
```

Comment:
I actually meant the whole DEFAULT_VALUE in defaults.get(foo, DEFAULT_VALUE) bit. It's fine. The cloud profiles stuff still needs to be ported to the new style of lookup tables.